routers/api/v1/admin/org.go

```diff
@@ -28,6 +28,10 @@ func CreateOrg(ctx *context.APIContext, form api.CreateOrgOption) {
 	//   description: username of the user that will own the created organization
 	//   type: string
 	//   required: true
+	// - name: organisation
```

Comment:
organisation or organization ?

Reply:
Sorry, I've fixed that. I presume we're standardising on US spelling rather than UK or International spelling...